qb_upload_limit_per_day.py

```diff
@@ -19,12 +19,92 @@
 current_session_previous_day_upload_data_usage = 0.0
 qb_online_status = False
 
+def login():
+    """
+    Performs login to qBittorrent WebUI using data from secrets.json
+    If login is successful, the resulting cookies are stored in 
+    cookies.txt file, so that the other methods can access them 
+    for authentification
+    """
+    file_name = "secrets.json"
+    if not exists(file_name):
+        return False, f"{file_name} with your username and password doesn't exist."
+    try:
+        with open(file_name) as f:
+            secrets = json.load(f)
+            if "username" not in secrets:
+                return False, f"username field is not in {file_name}"
+            if "password" not in secrets:
+                return False, "password field is not in {file_name}"
+    except Exception as e:
+        return False, f"Error {e} occured while reading JSON"
+
+    response = requests.get(f"{QB_URL}/api/v2/auth/login",params={"username":secrets["username"],"password":secrets["password"]})
```

Comment:
```suggestion
    response = requests.post(f"{QB_URL}/api/v2/auth/login",data={"username":secrets["username"],"password":secrets["password"]})
```

isn't `/api/v2/auth/login` a `POST` method? Btw I'm on `v5.0.0`. What version are you on?
![image](https://github.com/user-attachments/assets/c943eaf0-71d1-44c9-9fa3-6ad34d14b9a4)

Reply:
It is a bit of grey area, as we are not really modifing the state, just requesting a token. qBittorrent allows both:
![image](https://github.com/user-attachments/assets/add18316-a829-4e6a-813f-4021cec77d68)
But in general, POST is better, as the data is encrypted with HTTPS. I accidentally left is as GET, as it worked😀